### PR TITLE
IE11 Support - Note and sample fixes

### PR DESCRIPTION
**Pull request by AlexJerabek**
Tagging @ElizabethSamuel-MSFT for the writer review and @peronald to confirm the accuracy of the IE11 statement.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[264c699](https://github.com/OfficeDev/office-scripts-docs/commits/264c699f01864d61e1614b0fc926461b413b839b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/code-editor-environment.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-IE11/docs/overview/code-editor-environment.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-IE11/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C10%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202007101926062660-146%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2ByRGZcOmBO%2Fzd0LPR6PfVXpyCeIc3gdEYeT2N%2Fl6u8g%3D&st=2020-07-10T19%3A22%3A19Z&se=2020-08-10T19%3A27%3A19Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/overview/code-editor-environment.md [hunk 1]
```diff
@@ -32,6 +32,10 @@ The Excel Code Editor uses the same IntelliSense engine as Visual Studio Code. T
 
 Office Scripts do not support the usage of external, third-party JavaScript libraries. You are currently unable to call any library other than the Office Scripts APIs from a script. You do still have access to any [built-in JavaScript object](../develop/javascript-objects.md), such as [Math](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Math).
 
+## Browser support
+
+Office Scripts work in any browser that [supports Office for the web](https://support.microsoft.com/office/ad1303e0-a318-47aa-b409-d3a5eb44e452). However, some JavaScript features aren't supported in Internet Explorer 11. Any features introduced in [ES6 or later](https://www.w3schools.com/Js/js_es6.asp) won't work with Internet Explorer 11. If you have people in your organization still using that browser, be sure to test your scripts in that environment when sharing them.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/code-editor-environment.md:37]:
```suggestion
Office Scripts work in any browser that [supports Office for the web](https://support.microsoft.com/office/ad1303e0-a318-47aa-b409-d3a5eb44e452). However, some JavaScript features aren't supported in Internet Explorer 11 (IE 11). Any features introduced in [ES6 or later](https://www.w3schools.com/Js/js_es6.asp) won't work with IE 11. If people in your organization still use that browser, be sure to test your scripts in that environment when sharing them.
```

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT. Suggestion taken! I'll check in once I get sign off from someone a feature crew member.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b4b5a16](https://github.com/OfficeDev/office-scripts-docs/commits/b4b5a1603c4a3f624e87d90d15e44ca8ae499c41)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/code-editor-environment.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-IE11/docs/overview/code-editor-environment.md) | :white_check_mark:Succeeded | |
[docs/resources/scenarios/analyze-web-downloads.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-IE11/docs/resources/scenarios/analyze-web-downloads.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C10%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202007102041163094-146%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2DmwFcxmE4optufEMhQ65QPsh%2F5VE1b2kZS6SOz4uHs%3D&st=2020-07-10T20%3A37%3A30Z&se=2020-08-10T20%3A42%3A30Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**